src/components/Microsoft.DotNet.UpgradeAssistant.Extensions/ExtensionOptions.cs

```diff
@@ -18,5 +19,7 @@ public class ExtensionOptions
         public ICollection<ExtensionInstance> Extensions { get; } = new List<ExtensionInstance>();
 
         public Version CurrentVersion { get; set; } = null!;
+
+        public string ConfigurationFilePath { get; } = Path.Combine(Environment.CurrentDirectory, "upgrade-assistant.json");
```

Comment:
Would there be a need to add this as a user option?

Reply:
I don't see any scenario that would make sense. We may want to pivot it off of the directory of the input file, though